spec/services/users/delete_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Users::Delete, type: :service do
   before { omniauth_mock_github_payload }
 
-  let(:user) { create(:user, :with_identity, identities: ["github"]) }
+  let(:user) { create(:user, :trusted, :with_identity, identities: ["github"]) }
```

Comment:
These tests contain a lot of magical reflection built by @lightalloy, instead of creating a "if association is rating votes do something slightly different" I just decided to upgrade power up the user so that the rating vote validator will let them create a row in the DB